test/lib/mayaUsd/render/mayaToHydra/testStandardSurface.py

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright 2024 Autodesk, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import maya.cmds as cmds
+import fixturesUtils
+import mtohUtils
+import mayaUtils
+import unittest
```

Comment:
minor nitpick

do we need this?

Reply:
unittest is not needed, we can remove it